app/controllers/sign_up/registrations_controller.rb

```diff
@@ -18,6 +18,7 @@ def new
       @register_user_email_form = RegisterUserEmailForm.new
       session[:sign_up_init] = true
       analytics.track_event(Analytics::USER_REGISTRATION_ENTER_EMAIL_VISIT)
+      render :new, locals: { request_id: nil }
```

Comment:
That doesn't seem to be the case for `nil`:
```html
<input class="block col-12 field hidden" type="hidden" name="user[request_id]" id="user_request_id">
```

It works either way, but I think `nil` conveys the intent better, which is that by default, we don't want any value at all, not even an empty string, because it can imply presence. For example, if we had a conditional in the view that asked `if request_id`, then it would pass with an empty string, but not if it was `nil`.